tsconfig.json

```diff
@@ -21,10 +21,10 @@
     "skipLibCheck": true,
     "strictNullChecks": true,
     "suppressImplicitAnyIndexErrors": true,
-    "types": ["node"],
-    "baseUrl": "./src/",
+    "types": ["jest", "node"],
+    "baseUrl": "src",
     "outDir": "./build"
   },
-  "include": ["./src/**/*"],
+  "include": ["src"],
```

Comment:
removing will allow the `baseURL` import to work outside of `src`

Reply:
just filter out the rest via `exclude`

From ts:
```
Files included using "include" can be filtered using the "exclude" property. However, files included explicitly using the "files" property are always included regardless of "exclude". The "exclude" property defaults to excluding the node_modules, bower_components, jspm_packages and <outDir> directories when not specified.
```

Reply:
It seems there's several typescript problems, I'll create an issue just for that and apply a workaround so I can merge this PRs that are base for next ones: 
* https://github.com/gnosis/dex-telegram/issues/14